Derpitcraft/README.md

```diff
@@ -1,105 +1,65 @@
-
 ### Detailed Rules and Guidelines
-
-1. **No Griefing Policy**: Griefing is not tolerated at all. If you are caught, you will be **banned** from Derpitcraft Season 9 indefinitely. Our moderators will decide a suitable punishment. Griefing also includes stealing items from other players, destroying their bases, structures, and farms, and causing deaths / killing players (The PvP policy is explained in **Part 2**). It includes *harmful* pranking. Read more about the pranking policy in **Part 3**. For a better idea on what consists of a player's base, structure, or farm, see **Part 4**.
-
-  > To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.
-
+1. **No Griefing Policy**: Griefing is not tolerated at all. If you are caught, you will be **banned** from Derpitcraft Season 9 indefinitely. Our moderators will decide a suitable punishment. Griefing also includes stealing items from other players, destroying their bases, structures, and farms, and causing deaths / killing players (The PvP policy is explained in **Point 2**). It includes *harmful* pranking. Read more about the pranking policy in **Point 3**. For a better idea on what consists of a player's base, structure, or farm, see **Point 4**.
+To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.
 2.  **PvP Policy**: PvP is not permitted on the server except under specific conditions. PvP includes killing a player's pet, and damaging the player in any way, including but not limited to: using TNT, using weapons, using bows, using crossbows, using potions, and using pets. 
-
     - *PvP is permitted for self-defense*: If someone else attacks first, you can PvP to defend yourself. Don't escalate a fight for self-defense by using a better weapon. This is likely to make things worse. If it is a fist fight, try to resolve it peacefully without weapons at all. We do not recommend PvP to defend your base, structures, or farms. If someone is actively griefing your base, report the player *immediately* (see **Reporting**), and try to make the other party stop using the *chat*.
    
     - *PvP is permitted with consent*: If two or more players agree to PvP, they can PvP based on their own internal terms. We will not intervene unless a player is reported or griefing is detected. If both parties agree, you can have playful fights, kill the other party, and even make arena's for the entire server to enjoy.
    
-    - PvP fights and discrepancies are judged on a case to case basis, and an investigation will take place if a report is raised. This policy extends to harming players in any way.
-
+PvP fights and discrepancies are judged on a case to case basis, and an investigation will take place if a report is raised. This policy extends to harming players in any way.
```

Comment:
Markdown won't render this separately, add new line between paragraphs, also indent in to make markdown understand its part of the point.
```suggestion
    PvP fights and discrepancies are judged on a case to case basis, and an investigation will take place if a report is raised. This policy extends to harming players in any way.
```